README.md

```diff
@@ -44,6 +44,75 @@ They implement the following public methods:
 
 > **_NOTE:_**  See [SqsPermissionPublisher.ts](./packages/sqs/test/publishers/SqsPermissionPublisher.ts) for a practical example.
 
+#### PublisherManager
+
+PublisherManager is a wrapper to automatically spawn publishers on demand and reduce the amount of boilerplate needed to write them, while guaranteeing type-safety and autocompletion. It also automatically fills metadata fields.
+
+You can create an instance of PublisherManager as shown in this example:
+
+```typescript
+const diConfig: DiConfig = {
```

Comment:
I would remove DiConfig from the example, users of the library may not be using our template or awilix altogether